proto/observer/events.proto

```diff
@@ -19,3 +19,11 @@ message EventKeygenBlockUpdated {
   string keygen_block = 2;
   string keygen_pubkeys = 3;
 }
+
+message EventNewObserverAdded {
+  string msg_type_url = 1;
+  string observer_address = 2;
+  string zetaclient_grantee_address = 3;
```

Comment:
I may see that `GetAddressFromPubkeyString` has been added in your new commit.

If we have this method, do we still need to keep the address here then?

Reply:
I think we can keep it for the event . It might be easier for some consumers to filter using their addresses directly. 
don't have strong opinions about it though, that do you think?

Reply:
Sorry, haven't seen this is the event, this makes sense.